src/codemagic/tools/app_store_connect/actions/fetch_signing_files_action.py

```diff
@@ -218,7 +207,7 @@ def _create_missing_profiles(
         platform: Optional[BundleIdPlatform] = None,
     ) -> Iterator[Profile]:
         if not bundle_ids_without_profiles:
-            return []
+            return
```

Comment:
That was suddenly picked up by `mypy`. This is a generator function and return values are meaningless as the function does not return a single value at any point, but a generator object. This return just ensures that we're never going to reach a `yield` statement.